signer/test/test_user.py

```diff
@@ -0,0 +1,112 @@
+import os
+import unittest
+from tempfile import TemporaryDirectory
+
+import click
+from securesystemslib.signer import HSMSigner, SSlibKey
+
+from tuf_on_ci_sign._user import User
+
+# Long lines are ok here
+# ruff: noqa: E501
+
+REQUIRED = """
+[settings]
+pykcs11lib = /usr/lib/x86_64-linux-gnu/libykcs11.so
+user-name = @signer
+push-remote = origin
+pull-remote = myremote
+"""
+
+MISSING_NAME = """
+[settings]
+pykcs11lib = /usr/lib/x86_64-linux-gnu/libykcs11.so
+push-remote = origin
+pull-remote = myremote
+"""
+
+REQUIRED_AND_SIGNING_KEYS = """
+[settings]
+pykcs11lib = /usr/lib/x86_64-linux-gnu/libykcs11.so
+user-name = @signer
+push-remote = origin
+pull-remote = myremote
+
+[signing-keys]
+762cb22caca65de5e9b7b6baecb84ca989d337280ce6914b6440aea95769ad93 = hsm:2?label=YubiKey+PIV+%2315835999
+01ba4719c80b6fe911b091a7c05124b64eeece964e09c058ef8f9805daca546b = file:keys/mykey?encrypted=false
```

Comment:
Is the file signer URI ignored in the tests below on purpose?

Reply:
not really, same story as sigstore: I didn't think it was worth it (but wanted an example with more than one key)

I was planning to add it now but since sigstore is not possible I won't bother with this either